v2/datastream-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/datastream/ChangeEventConvertor.java

```diff
@@ -130,7 +132,7 @@ static Mutation.WriteBuilder changeEventToShadowTableMutationBuilder(
       Set<String> requiredKeyColumnNames = new HashSet<>(keyColumnNames);
       Mutation.WriteBuilder builder = Mutation.newInsertOrUpdateBuilder(shadowTableName);
 
-      populateMutationBuilderWithEvent(
+      ChangeEventSpannerUtils.populateMutationBuilderWithEvent(
```

Comment:
Why did we need to refactor these out into a `utils` class?  `Utils` class become very random with time as all sorts of mixed functionality is dumped into them...

Reply:
This is a class containing methods involving changeEvent and spanner structure like `com.google.cloud.spanner.Mutation`, `com.google.cloud.spanner.Key` etc. This is also something used by the bulk template hence was moved out. Maybe a better name is ChangeEventSpannerConvertor?